simplified-app-shared/src/main/java/org/nypl/simplified/app/catalog/CatalogBookReadController.kt

```diff
@@ -29,96 +31,114 @@ import org.slf4j.LoggerFactory
  */
 
 class CatalogBookReadController(
-  val activity: AppCompatActivity,
-  val analytics: AnalyticsType,
-  val profile: ProfileReadableType,
-  val account: AccountType,
-  val id: BookID,
-  val entry: FeedEntryOPDS) : OnClickListener {
+        val activity: AppCompatActivity,
```

Comment:
I think we're mashing each other's whitespace. Let's sort out an editor config that we can all import so that we're all using the same settings. I don't have any particular preference for formatting, but the codebase is currently using two spaces instead of four, so that might be the path of least resistance.